code/script.js

```diff
@@ -1,11 +1,13 @@
-// All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const nameForm = document.getElementById('name-form')
+const inputWrapper = document.getElementById('input-wrapper')
+const value = document.getElementById('name-input').value;
 
-// Global variables, if you need any, declared here
+let pickedCoffee 
+let pickedSize
+let step = 1;
```

Comment:
I can see that you implement a step counter and even you are increasing it in every function, but I can't see where you are using it. 

Probably you changed vision of how to handle passing from message to message and forgot to remove it 😸 
Remember to always clean your code from unused blocks of code 😄